internal/epoch/epoch_manager.go

```diff
@@ -1015,16 +1019,28 @@ func rangeCheckpointBlobPrefix(epoch1, epoch2 int) blob.ID {
 	return blob.ID(fmt.Sprintf("%v%v_%v_", RangeCheckpointIndexBlobPrefix, epoch1, epoch2))
 }
 
+func allowWritesOnIndexLoad() bool {
```

Comment:
Note: this env variable mechanism to control the behavior is temporary.
We will remove it once the corresponding code paths are _explicitly_ hooked in maintenance.